### PR TITLE
Upgrade dependency: ganache-core@2.11.2

### PR DESCRIPTION
**Pull request by CruzMolina**
* @truffle/artifactor: 2.10.2 →  2.11.2
* @truffle/contract: 2.10.2 →  2.11.2
* @truffle/core: 2.10.2 →  2.11.2
* @truffle/debugger: 2.10.2 →  2.11.2
* @truffle/deployer: 2.10.2 →  2.11.2
* @truffle/environment: 2.10.2 →  2.11.2
* @truffle/hdwallet-provider: 2.10.2 →  2.11.2
* @truffle/interface-adapter: 2.10.2 →  2.11.2
* @truffle/provider: 2.10.2 →  2.11.2
* truffle: 2.10.2 →  2.11.2

### CODE REVIEW

File: packages/interface-adapter/test/quorum-getBlock.test.ts [hunk 1]
```diff
@@ -20,9 +20,7 @@ async function prepareGanache(
     const server = Ganache.server({
       time: genesisBlockTime
     });
-    server.listen(port, (err: Error) => {
-      if (err) reject(err);
-
+    server.listen(port, () => {
```
Comment (haltman-at) [inline packages/interface-adapter/test/quorum-getBlock.test.ts:23]: Why was this removed?  I'm not familiar with this code so it's not obvious to me.

Comment (CruzMolina) [reply packages/interface-adapter/test/quorum-getBlock.test.ts:23]: It looks like this typing changed in the latest minor release of `ganache-core`: https://github.com/trufflesuite/truffle/runs/951396343#step:6:371

Comment (haltman-at) [reply packages/interface-adapter/test/quorum-getBlock.test.ts:23]: Oh weird.  Huh.  OK then!
